demo/src/example/example-page.js

```diff
@@ -8,67 +8,215 @@ import router from '../core/router';
 import { parseHtml } from '../core/html-utils';
 import Examples from './examples';
 import { openPlayerModal } from '../player/player-dialog';
+import Drm from '../../../src/utils/Drm';
```

Comment:
Another small change, can we remove the reference to this class since we are only pointing to a const?

Reply:
Done.